### ISSUE TITLE
Documentation incorrectly references npm package

### ISSUE THREAD

**Issue by EwanRoycroft** [Smaug6739/web-pinger#1]
The name of the package is "website-pinger", but the documentation consistently refers to it as "web-pinger".

This leads to some confusion when installing the package:
```
$ npm install web-pinger
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/web-pinger - Not found
npm ERR! 404 
npm ERR! 404  'web-pinger@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

**Comment by Smaug6739** [comment]
Thanks for the report il will fix that

**Comment by EwanRoycroft** [comment]
Fixed by 220633a9433b8a7f8f07c3a4326676b786f0a90e. Thanks 👍🏻
